deploy/galaxy/main.tf

```diff
@@ -215,7 +215,7 @@ module "saturn_execute" {
 
   max_instances = var.max_execute_instances
   min_instances = 0
-  load_ratio    = 10
+  load_ratio    = 5
```

Comment:
what does load_ratio do exactly? (can't find docs...)
and in particular are you sure you want to increase one and decrease the other?

Reply:
Not in docs because it's defined by us :')

We define it to be the number of jobs in queue per machine. This change halves the number of compile machines while doubling the number of execute machines.